client/app/scripts/reducers/__tests__/root-test.js

```diff
@@ -64,7 +64,7 @@ describe('RootReducer', () => {
       }
     ],
     stats: {
-      edge_count: 319,
+      edge_count: 379,
```

Comment:
I increased the constant in `graphExceedsComplexityThresh` so the definition of a *complex* graph changed.